src/persist-client/src/cfg.rs

```diff
@@ -416,6 +419,27 @@ pub const TXN_USE_CRITICAL_SINCE: Config<bool> = Config::new(
     "Use the critical since (instead of the overall since) when initializing a subscribe.",
 );
 
+/// Migrate the txns code to use the critical since when opening a new read handle.
```

Comment:
Looks like the rustdocs here are copy-pasta?

Reply:
Oops!